src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    public int year;
```

Comment:
The field 'year' should be private to adhere to encapsulation principles. Public fields can be accessed and modified directly, which can lead to unexpected behavior and make the class harder to maintain.